script-test/playbackstrategies/msestrategytest.js

```diff
@@ -765,25 +763,20 @@ require(
             expect(timeUtilsMock.calculateSlidingWindowSeekOffset).toHaveBeenCalledTimes(1);
           });
 
-          it('should autoresume when paused and seeking to the start of the sliding window seekable range', function () {
-            mockDynamicWindowUtils.shouldAutoResume.and.returnValue(true);
+          it('should start auto resume timeout when paused and seeking', function () {
             mockDashInstance.isPaused.and.returnValue(true);
 
-            mseStrategy.pause();
-            mseStrategy.setCurrentTime(0);
```

Comment:
Feel like we should keep these 2 lines in for it to be representative. `seeked` wouldn't get fired for no reason like it appears to be here.